amazon-kinesis-connector-flink/src/main/java/software/amazon/kinesis/connectors/flink/util/KinesisStateUtil.java

```diff
@@ -0,0 +1,33 @@
+package software.amazon.kinesis.connectors.flink.util;
+
+import org.apache.flink.api.common.ExecutionConfig;
+import org.apache.flink.api.common.typeinfo.TypeInformation;
+import org.apache.flink.api.common.typeutils.TypeSerializer;
+import org.apache.flink.api.java.tuple.Tuple2;
+import org.apache.flink.api.java.typeutils.runtime.TupleSerializer;
+import org.apache.flink.api.java.typeutils.runtime.kryo.KryoSerializer;
+import software.amazon.kinesis.connectors.flink.model.SequenceNumber;
+import software.amazon.kinesis.connectors.flink.model.StreamShardMetadata;
+
+/**
+ * Utilities for Flink Kinesis connector state management.
+ */
+public class KinesisStateUtil {
```

Comment:
Since this is a static util class please add a private constructor to prevent instantiation

Reply:
Done